QuickDeployment/QuickDeploymentClientSetup.md

```diff
@@ -2,41 +2,52 @@
 
 <!-- TOC -->
 
-- [Summary](#summary)
-- [DNS](#dns)
-- [Client Installation](#client-installation)
+- [Quick Deployment Client Setup](#quick-deployment-client-setup)
+    - [Summary](#summary)
+    - [DNS](#dns)
+    - [Client Installation](#client-installation)
 
 <!-- /TOC -->
 
 ## Summary
+
 Once you have QDE set up in your environment, you'll need to get clients talking to it. To do that, we'll need to do the following:
 
-* Setting up DNS on the client to access QDE.
-* Install the QDE SSL/TLS certificate so clients can access HTTPS components
-* Install Chocolatey and friends
+- Setting up DNS on the client to access QDE.
+- Install the QDE SSL/TLS certificate so clients can access HTTPS components
+- Install Chocolatey and friends
 
 ## DNS
-Typically in your environment, onces you've added QDE, it should be able to start talking to the QDE. In some situations, you may need to add the host name with the IP address to reach your environment.
+
+Typically in your environment, onces you've added QDE, it should be able to start talking to the QDE.
+In some situations, you may need to add the host name with the IP address to reach your environment.
 
 ## Client Installation
 
 On your client machines, you will be running the following script in an administrative context:
 
 ```powershell
-Set-ExecutionPolicy RemoteSigned -Scope Process -Force; iex ((New-Object System.Net.WebClient).DownloadString('https://chocolatey.org/Import-QuickDeployCertificate.ps1')); Set-ExecutionPolicy RemoteSigned -Scope Process -Force; iex ((New-Object System.Net.WebClient).DownloadString('https://chocoserver:8443/repository/choco-install/ClientSetup.ps1'))
+Set-ExecutionPolicy RemoteSigned -Scope Process -Force
+iex ((New-Object System.Net.WebClient).DownloadString('https://chocolatey.org/Import-QuickDeployCertificate.ps1'))
+Set-ExecutionPolicy RemoteSigned -Scope Process -Force
+iex ((New-Object System.Net.WebClient).DownloadString('https://chocoserver:8443/repository/choco-install/ClientSetup.ps1'))
 ```
```

Comment:
This was put in the way it was for a reason - not keen on making this change.

Reply:
Can you elaborate on that reason?

Reply:
One liner for people to copy versus several lines. You would be surprised, or maybe not, that folks can mess up the multi-liner. I've seen it many a time 🤦, so we provide one liners.